_extras/design.md

```diff
@@ -0,0 +1,126 @@
+    *   Key decisions
+        1.  What is the structure of the data/results to communicate?
+            *   Relationship
+            *   Comparison
+            *   Distribution
+            *   Composition
+        2.  How many variables or categories need to be conveyed?
+            *   Is one of those variables time?
+        3.  How much data is available for each variable or category?
+        4.  Are absolute or relative values more appropriate to communicate?
+2.  Learners will be able to implement the appropriate graphical device using `dplyr` and `ggplot2`.
+    *   Basic graphing layers
+        *   `ggplot()` 
+        *   `aes()`
+        *   `geom_line()`
+        *   `geom_bar()`
+        *   `geom_smooth()`
+        *   `geom_point()`        
+        *   `geom_boxplot()`
+        *   `geom_smooth()`
+    *   Facet subplots
+        *   `facet_grid()`
+        *   `facet_wrap()`
+    *   Dataset and aesthetic adjustments (*Publication quality figures*)
+        *   `scale_manual()`
+        *   `lims()`
+        *   `labs()`
+        *   `guide_legend()`
+        *   `theme()`        
+
+### Summative Assessment
+
+*   Midpoint: Given a research question and some data, choose and generate the best chart.
+*   Final: Evaluate an analysis report generated from code that includes four or five pairs of research questions and charts:
+    1.  Modify some data that is laid out differently (*but is still tabular*) than what is required by the graphing process
+    2.  Find and fix a chart that is not the best option to address the research question
+    3.  Modify one chart to be publication quality
+
+### Essential Questions
+
+How do I...
+
+*   Read, analyze, and visualize a tabular data set?
+*   Manipulate, summarize, and analyze data to answer a research question?
+*   Choose the best chart to answer my research question?
+*   Generate the best chart to answer my research question?
+*   Produce a publication quality version of my chart?
+
+### Learners Will Be Able To...
+
+*   Interactively generate analysis code and save it in a file
+*   Read tabular data into a data frame
+*   Manipulate and summarize tabular data using `dplyr`
+*   Generate publication quality charts using `ggplot2`
+*   Critique others' choices of charts and esthetics and express how to improve them
+*   Access `help()`, vignette tutorials, and online documentation
+*   Interpret common error/warning messages from `dplyr` and `ggplot2`
+
+### Learners Will Know...
+
+*   That a clearly defined research question is essential to a successful analysis and graphical representation of the results
+*   Where to find documentation on `dplyr` and `ggplot2`
+*   How to find out what other graphical software R offers
+
+## Stage 2 - Learning Plan
+
+*To Do: point-form outline of the lesson episodes with time estimates for teaching and exercises.*
```

Comment:
Good - I'll resurrect the point-form outline we did for the Python gapminder lesson and send it to you as an example.